### PR TITLE
Awesome book using Class

### PR DESCRIPTION
**Pull request by Kandy-Peter**
This pull request contains the following steps and elements

-create a new js file
- Use class instead of a function to get and display data, create and remove books
- Add some styles to the template.
- fix bugs and Linters errors

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+let bookArray = [];
```
Comment (leolpaz) [inline index.js]: - [x] As pointed in the requirement screenshotted below, arrays should be inside the class, kindly refactor the code to do that
image

File: index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  }
+});
+
+booksContainer.addEventListener('click', (e) => {
+  if (e.target.id === 'remove') {
+    const bookItem = e.target.parentElement;
+    booksContainer.removeChild(bookItem);
+    const newBookArray = bookArray.filter((event) => event.id !== bookItem.id);
+    bookArray = newBookArray;
+    localStorage.setItem('bookstore', JSON.stringify(bookArray));
```
Comment (leolpaz) [inline index.js]: - [ ] As pointed in the requirement below, kindly refactor your remove, to be a class method
image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @Kandy-Peter , 

You've done an amazing job so far, I'd like to highlight the following:
- Great job with the add book method
- Linter is passing

Some issues need to be resolved before your approval, but you're almost there!




## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — COMMENTED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Kandy-Peter ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**